programs/create-genesis/ee_genesis/event_engine_genesis.hpp

```diff
@@ -14,12 +14,15 @@ class event_engine_genesis final {
 
     void start(const bfs::path& ee_directory, const fc::sha256& hash);
     void finalize();
-public:
-    ee_genesis_serializer messages;
-    ee_genesis_serializer transfers;
-    ee_genesis_serializer pinblocks;
-    ee_genesis_serializer accounts;
-    ee_genesis_serializer funds;
+
+    enum ee_ser_type {messages, transfers, pinblocks, accounts, funds, balance_conversions};
+    ee_genesis_serializer& get_serializer(ee_ser_type type) {
```

Comment:
It can be even more improved if don't store serializers in fields at all, but create serializer when it need, and close opened section in serializer's destructor. And `event_engine_genesis::finalize` willn't be need. Additionally, creating new serializer should destroy previous to correct close if two serializers is created in same { } scope.
All of it is useful to correct order of "Finish section" logs messages.
But it is not required in this PR.